cms/static/cms/js/libs/pep.js

```diff
@@ -0,0 +1,1330 @@
+/*!
```

Comment:
even though they are libs and in the end it gets packaged, I'll recommend adding it minified. Just to make sure that no code gets changed accidently.